pydantic/fuzz_parse.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python3
```

Comment:
why do we need this file? Surely the source shouldn't have to change.

Reply:
This file is the fuzz harness/test itself. The fuzz engine - [Atheris](https://github.com/google/atheris) - will run this test repeatedly.

This PR does not alter the pydantic source code but adds a fuzz test. The fuzzer can be placed anywhere in this repository though.

Reply:
Okay, well it definitely shouldn't be in `pydantic`, best to put it in `tests`